src/main/java/gift/controller/ProductController.java

```diff
@@ -0,0 +1,66 @@
+package gift.controller;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import gift.Model.Product;
+
+@RestController
+@RequestMapping("/api/products")
+public class ProductController {
+    private final Map<Long, Product> products = new HashMap<>();
+
+    @GetMapping("")
```

Comment:
path가 비어있는 경우 생략 가능합니다 :) 

```suggestion
    @GetMapping
```

Reply:
넵!